README.md

```diff
@@ -54,6 +54,85 @@ test.all(common)
 
 A valid (read: that follows this interface) IPFS core implementation, must expose the following API.
 
+## Group Files
```

Comment:
Is "Group Files" a canonical api section name? I haven't seen this referred to anywhere -- could you link me?